pom.xml

```diff
@@ -68,7 +68,7 @@
 		<dependency>
 			<groupId>com.h2database</groupId>
 			<artifactId>h2</artifactId>
-			<scope>runtime</scope>
+			<!-- <scope>runtime</scope> -->
```

Comment:
I think this can stay, but could you leave a comment for why it would or wouldn't be commented out?

Reply:
The h2-console can not be accesses if the scope is runtime Commenting it would will make it default till console isn't needed